### DISCUSSION TITLE
epoptes Label IP

### DISCUSSION THREAD

**Discussion by awanbandung** [epoptes/epoptes#199] [Q&A] [answered]
currently i using epoptes for LTSP-server client, but it show label computer (user) , its possible to show label with IP Address?

because i running 1 server with multiple network IP (VLAN)

**Comment by alkisg** [comment]
Hi, no currently this isn't possible.
Why isn't the client hostname suitable, even if you have multiple VLANs?
E.g. you can name your LTSP clients like `a01`, `b12` or `c07`, where a, b, c are the VLAN names and 01..12 is the PC number.

**Reply by awanbandung** [reply]
oo u right, so i just edit at ltsp.conf ?


```
[clients]
# Specify an /etc/fstab line for NFS home; note this is insecure
# FSTAB_HOME="server:/home /home nfs defaults,nolock 0 0"

HOSTNAME=LTSP-2-2
AUTOLOGIN=user1
PRE_INITRD_BOTTOM_IMAGE_TO_RAM="dd if=/root/images/x86_64.img of=/dev/null &"
#IMAGE_TO_RAM=1
```

how to config 2 hostname with 2 VLAN  at ltsp.conf?

**Comment by alkisg** [comment]
Just put the mac addresses as sections, and the HOSTNAMEs under them. Remember to run `sudo ltsp initrd` and to reboot the clients:
```
[a1:02:03:04:05:06]
HOSTNAME=a01

[b1:02:03:04:05:07]
HOSTNAME=b02
```

**Reply by awanbandung** [reply]
that;s a MAC address of ltsp server ? I try, but client still with default  hostname like ltsp47 

```

[06:42:65:82:0b:bb]
HOSTNAME=102
AUTOLOGIN=user1
PRE_INITRD_BOTTOM_IMAGE_TO_RAM="dd if=/root/images/x86_64.img of=/dev/null &"
RELOGIN=1
#GDM3_CONF="WaylandEnable=false"
#LIGHTDM_CONF="greeter-hide-users=false"
SDDM_CONF="/etc/ltsp/sddm.conf"
CUPS_SERVER="$SERVER"
DISABLE_SYSTEM_SERVICES="wazuh-agent suricata"
```

**Comment by alkisg** [comment, marked as answer]
No, these are the mac addresses of the clients.

**Comment by awanbandung** [comment]
> No, these are the mac addresses of the clients.

ooo i think, i see,  ty for explanation, maybe i will use same hostname
